reliability/README.md

```diff
@@ -128,13 +132,52 @@ Set environment virable SLACK_API_TOKEN before running the test. Contact qili@re
     # you must be a member of the slack channel to receive the notification.
     slack_member: <Your slack member id>
 ```
-
 In the above configuration, notifications will be sent to [#ocp-qe-reliability-monitoring](https://coreos.slack.com/archives/C0266JJ4XM5) (Channel ID: C0266JJ4XM5) in [CoreOS](coreos.slack.com) workspace, and @ the user of slack_member if you configured.
 
 If you're in [CoreOS](coreos.slack.com) workspace, but you want to use your own slack channel, create a slack channel and install App `OCP Reliability` which already exists in CoreOS workspace.
 
 If you want to use your own App(slack_api_token), create an [app](https://api.slack.com/apps?new_granular_bot_app=1) and add a bot to it on slack. Slack Bot Token Scopes permissions are [channels:read] [chat:write] [groups:read] [im:read] [mpim:read]. You will get a token after the app is installed to a workspace. Install the app to your channel. Set the token to SLACK_API_TOKEN environment variable.
 
+### Kraken Integration
+Configure Kraken scenario(s) to trigger error injection during the Reliability test.
+
+The below configuration example enables the Kraken integration by setting `kraken_enable: True`. 
+`kraken_scenarios` defines the [Kraken scenario(s)](https://github.com/cloud-bulldozer/kraken-hub/blob/main/README.md#supported-chaos-scenarios) you want to run during Reliability test. Refer to each Kraken scenario's document for the supported `parameters`, e.g [pod-scenarios](https://github.com/cloud-bulldozer/kraken-hub/blob/main/docs/pod-scenarios.md#supported-parameters).
```

Comment:
The link you have here isn't the exact names to enter for the kraken-hub scenarios. Might want to include the following list: 
pod-scenarios
node-scenarios
zone-outages
time-scenarios
power-outages
container-scenarios
node-cpu-hog
node-io-hog
node-memory-hog
namespace-scenarios
application-outages

Reply:
@paigerube14 Thanks for pointing out this. I'll update it according to https://github.com/cloud-bulldozer/kraken-hub/blob/main/README.md#supported-chaos-scenarios and point user to this link as there will always be the latest.